### PR TITLE
accepted revisions for SDL-0227 - Add Supported RGB Colors to Light Capabilities

### PR DESCRIPTION
**Pull request by yang1070**
Changes;
- Change RGBColorsSupported to RGBColorRange
- Change supportedRGBColors to supportedRGBColorRanges
- Update RGBColorRanges struct description to include the following cases:
  - If only minimums are included for R, G, and B, it is a single color
  - If a max is included on any R, G, B color, it is a range of color
  - If two or more colors contain both min and max, those two colors can be shaded together within the ranges supplied

### CODE REVIEW

**Comment by yang1070** [comment]
@theresalech this is ready for review. Thanks!

File: proposals/0227-add-supported-rgb-colors.md [hunk 1]
```diff
@@ -15,44 +15,40 @@ To avoid trial and error and allow mobile apps to know what RGB colors are suppo
...
+    <param name="redMin" type="Integer" minvalue="0" maxvalue="255" mandatory="true" />
+    <param name="redMax" type="Integer" minvalue="0" maxvalue="255" mandatory="false" />
+    <param name="greenMin" type="Integer" minvalue="0" maxvalue="255" mandatory="true" />
+    <param name="greenMax" type="Integer" minvalue="0" maxvalue="255" mandatory="false" />
+    <param name="blueMin" type="Integer" minvalue="0" maxvalue="255" mandatory="true" />
+    <param name="blueMax" type="Integer" minvalue="0" maxvalue="255" mandatory="false" />
 </struct>
 ```
 
+The changes to the HMI API are smiliar to those in the mobile API.
```
Comment (theresalech) [inline proposals/0227-add-supported-rgb-colors.md]:
```suggestion
The changes to the HMI API are similar to those in the mobile API.
```

Comment (yang1070) [reply proposals/0227-add-supported-rgb-colors.md]: updated

**Comment by yang1070** [comment]
@theresalech sorry for the typos. Now it is updated

**Pull request merged**